tools/version-bump/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Internal dependencies
+ */
+import { program } from './program';
+import './commands/bump';
+
+// Start the program
+program.parse();
```

Comment:
Pretty sure you need to pass `process.argv` here.

Reply:
I noticed `release-post-generator` didn't pass them in either, but [that doesn't seem to matter](https://github.com/tj/commander.js#common-option-types-boolean-and-value)

>program.parse(arguments) processes the arguments, leaving any args not consumed by the program options in the program.args array. The parameter is optional and defaults to process.argv.

I just passed them in anyways for better clarity.

Reply:
Oh you're right! I missed that it was optional!